tests/test_filter_distribcell/case-4/results_true.dat

```diff
@@ -0,0 +1,17 @@
+k-combined:
+0.000000E+00 0.000000E+00
+tally 1:
+2.166056E-02
```

Comment:
Why are there results for 14 tally bins here? I would have though there would only be 7 for the 7 pin cells in the problem.